lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -65,6 +74,10 @@ use tokio_boring::{HandshakeError, SslStream};
 
 const RDP_CONNECT_TIMEOUT: Duration = Duration::from_secs(5);
 
+struct ResizeWithholder {
```

Comment:
Maybe a comment on the purpose of this utility would help (ie why we need to withhold resizes)